packages/components/PersonaCoin/src/PersonaCoin.tokens.icon.ts

```diff
@@ -7,14 +7,21 @@ import { calculateEffectiveSizes } from './PersonaCoin.helpers';
 const _iconKeyProps: (keyof IPersonaCoinTokens)[] = ['iconSize', 'size', 'coinSize'];
 
 function _buildIconStyles(tokenProps: IPersonaCoinTokens /*, theme: ITheme */): ImageProps {
-  const { iconSize } = calculateEffectiveSizes(tokenProps);
+  const { iconSize, iconStrokeWidth } = calculateEffectiveSizes(tokenProps);
+  const iconSizeAdjusted = iconSize + iconStrokeWidth * 2;
+  const iconStrokeColor = tokenProps.iconStrokeColor || 'white';
 
   return {
     source: {},
     style: {
       position: 'absolute',
-      width: iconSize,
-      height: iconSize
+      width: iconSizeAdjusted,
+      height: iconSizeAdjusted,
+      bottom: -iconStrokeWidth,
```

Comment:
does negative values of bottom work in win32?

Reply:
I don't know, but if it didn't I think it would be a win32 bug.  It works on all other platforms including web.

Reply:
Could you test it on win32 and log a bug if it doesn't work? we don't want this bug to go unnoticed.

Reply:
I do not have access to a Windows machine.

Reply:
I can try checking this out on my win32 machine later today

Reply:
Seems to be working fine?

![Screen Shot 2020-08-31 at 3 26 02 PM](https://user-images.githubusercontent.com/6722175/91774729-56181380-eb9e-11ea-9772-a2d293f36228.png)

Reply:
Side note, I couldn't drag the sliders in the "Customize Usage" part. not sure if that's an existing bug

Reply:
Yes, I think it was broken a while ago. :(